### PR TITLE
Rewrite schema.yaml with un-nested comments

### PR DESCRIPTION
**Pull request by Polo2**
Purpose of this PR is to rewrite routes for `comments` , on `swagger.yaml`

### CODE REVIEW

File: doc/swagger.yaml [hunk 1]
```diff
@@ -86,7 +86,7 @@ paths:
           description: Invalid input
       security:
         - APIKeyHeader: []
-  '/dependencies/{dependencyId}/comments/{commentId}':
+  '/comments/{comments_id}':
```
Comment (michaelbaudino) [inline doc/swagger.yaml]: I think it should be `{comment_id}`, singular. Or even better: just `id` :trophy:

File: doc/api-specification.yaml [hunk 2]
```diff
@@ -112,6 +112,9 @@ paths:
           description: ID of dependency
           required: true
           type: integer
+      operationId: getComments
```
Comment (thibaultdalban) [inline doc/api-specification.yaml]: We do not use any more `operationId` you can remove it.

File: doc/api-specification.yaml [hunk 3]
```diff
@@ -112,6 +112,9 @@ paths:
...
+      produces:
```
Comment (thibaultdalban) [inline doc/api-specification.yaml]: This is already defined [here](https://github.com/alpinelab/toolbox/pull/30/files#diff-7af801d6120b8194457e2a609b00234eR21) for the entire specification, you do not need to add it.

File: doc/api-specification.yaml [hunk 4]
```diff
@@ -124,16 +127,12 @@ paths:
     post:
       tags:
         - comment
-      summary: create a comment
+      summary: Create comment
+      description: Add a new comment to dependency
+      operationId: addComment
```
Comment (thibaultdalban) [inline doc/api-specification.yaml]: Same thing here.

File: doc/api-specification.yaml [hunk 5]
```diff
@@ -142,6 +141,8 @@ paths:
                 $ref: '#/definitions/Comment/properties/body'
               published:
                 $ref: '#/definitions/Comment/properties/published'
+              dependency_id:
```
Comment (thibaultdalban) [inline doc/api-specification.yaml]: This should be in a dedicated PR, it is not related with the URL nesting. Same thing for the `summary` and `description` few lines above.

**Comment by Polo2** [comment]
Modifications about un-nesting comments have already been pushed on commit 9442f2801e26674fbdadf87f94ee715f72fa47ab
